# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../gcc/mips64.patch
# Copyright (C) 2010 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

We do not need 3 multilib configs by default, use (old) 32bit compatible
lib/ and true 64bit lib64/, like on x86-64, powerpc64, ...

  -R ene Rebe <rene@exactcode.de>

--- gcc-4.5.1/gcc/config/mips/linux64.h.orig	2010-09-23 20:31:19.054969380 +0000
+++ gcc-4.5.1/gcc/config/mips/linux64.h	2010-09-23 20:32:03.394976017 +0000
@@ -26,7 +26,7 @@
   BASE_DRIVER_SELF_SPECS, \
   LINUX_DRIVER_SELF_SPECS \
   " %{!EB:%{!EL:%(endian_spec)}}" \
-  " %{!mabi=*: -mabi=n32}"
+  " %{!mabi=*: -mabi=64}"
 
 #undef LIB_SPEC
 #define LIB_SPEC "\
--- gcc-4.5.1/gcc/config/mips/t-linux64.orig	2010-09-23 20:16:34.872442972 +0000
+++ gcc-4.5.1/gcc/config/mips/t-linux64	2010-09-23 20:17:39.415310440 +0000
@@ -16,9 +16,9 @@
 # along with GCC; see the file COPYING3.  If not see
 # <http://www.gnu.org/licenses/>.
 
-MULTILIB_OPTIONS = mabi=n32/mabi=32/mabi=64
-MULTILIB_DIRNAMES = n32 32 64
-MULTILIB_OSDIRNAMES = ../lib32 ../lib ../lib64
+MULTILIB_OPTIONS = mabi=64/mabi=32
+MULTILIB_DIRNAMES = 64 32
+MULTILIB_OSDIRNAMES = ../lib64 ../lib
 
 EXTRA_MULTILIB_PARTS=crtbegin.o crtend.o crtbeginS.o crtendS.o crtbeginT.o
 
